<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mri_rt_proj_Widget</class>
 <widget class="QWidget" name="mri_rt_proj_Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1440</width>
    <height>800</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Proj</string>
  </property>
  <widget class="QWidget" name="plotWidget" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>531</width>
     <height>841</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <layout class="QVBoxLayout" name="plotLayout"/>
  </widget>
  <widget class="QWidget" name="controlWidget" native="true">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>1000</x>
     <y>10</y>
     <width>361</width>
     <height>861</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>300</width>
     <height>0</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <widget class="QLabel" name="freqLabel">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>80</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Frequency/MHz</string>
    </property>
   </widget>
   <widget class="QPushButton" name="startButton">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>130</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="freqValue">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>80</y>
      <width>95</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="decimals">
     <number>5</number>
    </property>
    <property name="maximum">
     <double>60.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.001000000000000</double>
    </property>
    <property name="value">
     <double>15.670000000000000</double>
    </property>
   </widget>
   <widget class="QPushButton" name="stopButton">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>130</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Stop</string>
    </property>
   </widget>
   <widget class="QPushButton" name="acquireButton">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>290</y>
      <width>91</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Acquire</string>
    </property>
   </widget>
   <widget class="QLabel" name="ProjLabel">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>30</y>
      <width>221</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Projection control panel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="loadShimButton">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>700</y>
      <width>150</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>load shims</string>
    </property>
   </widget>
   <widget class="QPushButton" name="zeroShimButton">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>730</y>
      <width>150</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>zero shims</string>
    </property>
   </widget>
   <widget class="QDial" name="dial">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>190</y>
      <width>101</width>
      <height>91</height>
     </rect>
    </property>
    <property name="maximum">
     <number>360</number>
    </property>
    <property name="value">
     <number>90</number>
    </property>
    <property name="sliderPosition">
     <number>90</number>
    </property>
    <property name="invertedAppearance">
     <bool>true</bool>
    </property>
    <property name="invertedControls">
     <bool>false</bool>
    </property>
    <property name="wrapping">
     <bool>true</bool>
    </property>
    <property name="notchTarget">
     <double>45.000000000000000</double>
    </property>
    <property name="notchesVisible">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="numAvgLabel">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>371</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Number of averages</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="numAvg">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>370</y>
      <width>41</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="decimals">
     <number>0</number>
    </property>
    <property name="maximum">
     <double>60.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>1.000000000000000</double>
    </property>
    <property name="value">
     <double>1.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="shimmingLabel">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>424</y>
      <width>260</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Shimming tools(mA of current)</string>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider_x">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>457</y>
      <width>290</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-1000</number>
    </property>
    <property name="maximum">
     <number>1000</number>
    </property>
    <property name="value">
     <number>120</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="invertedAppearance">
     <bool>false</bool>
    </property>
    <property name="tickPosition">
     <enum>QSlider::TicksBelow</enum>
    </property>
    <property name="tickInterval">
     <number>50</number>
    </property>
   </widget>
   <widget class="QLabel" name="gradoffsetLabel_x">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>481</y>
      <width>150</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>X gradient offset</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="gradOffset_disp_x">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>484</y>
      <width>44</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>120</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="gradOffset_x">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>484</y>
      <width>60</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-1000</number>
    </property>
    <property name="maximum">
     <number>1000</number>
    </property>
    <property name="value">
     <number>120</number>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider_y">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>524</y>
      <width>290</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-1000</number>
    </property>
    <property name="maximum">
     <number>1000</number>
    </property>
    <property name="value">
     <number>45</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="invertedAppearance">
     <bool>false</bool>
    </property>
    <property name="tickPosition">
     <enum>QSlider::TicksBelow</enum>
    </property>
    <property name="tickInterval">
     <number>50</number>
    </property>
   </widget>
   <widget class="QLabel" name="gradoffsetLabel_y">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>547</y>
      <width>150</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Y gradient offset</string>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider_z">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>594</y>
      <width>290</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-1000</number>
    </property>
    <property name="maximum">
     <number>1000</number>
    </property>
    <property name="value">
     <number>-92</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="invertedAppearance">
     <bool>false</bool>
    </property>
    <property name="tickPosition">
     <enum>QSlider::TicksBelow</enum>
    </property>
    <property name="tickInterval">
     <number>50</number>
    </property>
   </widget>
   <widget class="QLabel" name="gradoffsetLabel_z">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>617</y>
      <width>150</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Z gradient offset</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="gradOffset_y">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>547</y>
      <width>60</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-1000</number>
    </property>
    <property name="maximum">
     <number>1000</number>
    </property>
    <property name="value">
     <number>45</number>
    </property>
   </widget>
   <widget class="QLineEdit" name="gradOffset_disp_y">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>548</y>
      <width>44</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>45</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="gradOffset_z">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>620</y>
      <width>60</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-1000</number>
    </property>
    <property name="maximum">
     <number>1000</number>
    </property>
    <property name="value">
     <number>-92</number>
    </property>
   </widget>
   <widget class="QLineEdit" name="gradOffset_disp_z">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>620</y>
      <width>44</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>-92</string>
    </property>
   </widget>
   <widget class="QPushButton" name="saveShimButton">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>670</y>
      <width>150</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>save shims</string>
    </property>
   </widget>
   <widget class="QPushButton" name="searchButton">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>330</y>
      <width>150</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Find angle</string>
    </property>
   </widget>
   <widget class="QLabel" name="angleLabel">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>220</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Angle</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="angleSpin">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>220</y>
      <width>60</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="maximum">
     <number>360</number>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QLineEdit" name="angleDisp">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>220</y>
      <width>44</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="maxAngleLabel">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>250</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Max</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="maxAngleDisp">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>250</y>
      <width>44</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QComboBox" name="acqType">
    <property name="geometry">
     <rect>
      <x>89</x>
      <y>293</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>770</y>
      <width>301</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QProgressBar {

border: 2px solid grey;

border-radius: 5px;

background-color: #FFFFFF;

}

QProgressBar::chunk {

background-color: #88B8D7;

width: 20px;

}

QProgressBar {

text-align: center;

color: rgb(0, 0, 0);

}</string>
    </property>
    <property name="value">
     <number>50</number>
    </property>
   </widget>
   <zorder>gradOffset_z</zorder>
   <zorder>angleSpin</zorder>
   <zorder>gradOffset_x</zorder>
   <zorder>freqLabel</zorder>
   <zorder>startButton</zorder>
   <zorder>freqValue</zorder>
   <zorder>stopButton</zorder>
   <zorder>acquireButton</zorder>
   <zorder>ProjLabel</zorder>
   <zorder>loadShimButton</zorder>
   <zorder>zeroShimButton</zorder>
   <zorder>dial</zorder>
   <zorder>numAvgLabel</zorder>
   <zorder>numAvg</zorder>
   <zorder>shimmingLabel</zorder>
   <zorder>horizontalSlider_x</zorder>
   <zorder>gradoffsetLabel_x</zorder>
   <zorder>gradOffset_disp_x</zorder>
   <zorder>horizontalSlider_y</zorder>
   <zorder>gradoffsetLabel_y</zorder>
   <zorder>horizontalSlider_z</zorder>
   <zorder>gradoffsetLabel_z</zorder>
   <zorder>gradOffset_y</zorder>
   <zorder>gradOffset_disp_y</zorder>
   <zorder>gradOffset_disp_z</zorder>
   <zorder>saveShimButton</zorder>
   <zorder>searchButton</zorder>
   <zorder>angleLabel</zorder>
   <zorder>angleDisp</zorder>
   <zorder>maxAngleLabel</zorder>
   <zorder>maxAngleDisp</zorder>
   <zorder>acqType</zorder>
   <zorder>progressBar</zorder>
  </widget>
  <widget class="QWidget" name="imageWidget" native="true">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>10</y>
     <width>400</width>
     <height>841</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>400</width>
     <height>16777215</height>
    </size>
   </property>
   <layout class="QVBoxLayout" name="imageLayout"/>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
